client/assets/js/apple.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+export default class Apple {
+    
+    constructor(x, y) {
+        this.x = x;
+        this.y = y;
+    }
+    
+    draw(context) {
```

Comment:
how often will it happen to draw the same apple in different contexts?
not so often.
The `context` should be passed to the constructor